static/app/views/insights/mobile/appStarts/components/samples.tsx

```diff
@@ -94,7 +94,7 @@ export function SamplesTables({transactionName}: any) {
             });
             setSampleType(value);
           }}
-          defaultValue={SPANS}
+          value={sampleType}
```

Comment:
we pass `onChange` here wer we set the `sampleType`, so we might just as well pass the same `sampleType` as `value`. 

The state is also initialized with `SPANS`:

```
const [sampleType, setSampleType] = useState<typeof EVENT | typeof SPANS>(SPANS);
```

so this works exactly the same as before.